index.html

```diff
@@ -0,0 +1,578 @@
+2. **must** match the default name of the spec, as derived from the spec's URL.
+
+With the exception of their root directive, specs **should** define any new schema elements with names that start with `$name__`. The root directive can simply be named with the name of the spec.
+
+```graphql
+#::[Using the `@core` directive with a spec's default name]
+schema @core(using: "https://spec.example.com/example/v1.0") {
+  query: Query
+}
+
+type User {
+  name: String @example(data: ITEM)
+}
+
+# An enum used to provide structured data to the example spec.
+# It is prefixed with the name of the spec.
+enum example__Data {
+  ITEM
+}
+
+directive @example(data: example__Data) on FIELD
+```
+
+The language of the spec **should** specify the default prefix and **should** identify locations where the prefix is used. The prefix **should not** be elided within documentation; definitions of schema elements provided within the spec **must** include the default prefix.
+
+### Elements which should be prefixed
+Specs **should** attach their prefix to:
+- the names of any object types, interfaces, unions, enums, or input object types defined in the spec
+- the names of any fields the spec introduces on *foreign types* defined in a different spec
+- the names of any arguments the spec introduces on *foreign directives and fields* defined in a different spec
+- the names of any directives introduced in the spec
+  - specs may introduce one **root directive**, which carries *only* the prefix and no underscore.
+
+<view-of src=prefixing.graphql
+  title='Prefixing examples'
+></view-of>
+
+## Versioning
+
+Specs are versioned with a **subset** of a [Semantic Version Number](https://semver.org/spec/v2.0.0.html) containing only the `major` and `minor` parts. Thus, specifications **should** provide a version of the form `‹major›.‹minor›`, where both integers >= 0.
+
+```
+#::[Examples of valid versions]
+2.2
+1.0
+1.1
+0.1
+```
+
+As specified by semver, spec authors **should** increment the:
+> - MAJOR version when you make incompatible API changes,
+> - MINOR version when you add functionality in a backwards compatible manner
+
+Patch and pre-release qualifiers are judged to be not particularly meaningful in the context of specifications, which are (by definition) interfaces rather than implementations. The `‹patch›` component of version denotes a bug fix which is backwards compatible—that is, a change to the implementation which does not affect the interface. `‹patch›`-level changes in the version of a spec denote wording clarifications which do not require implementation changes. As such, it is not important to track them for the purposes of version resolution.
+
+As with [semver](https://semver.org/spec/v2.0.0.html), the `0.x` version series is special: there is no expectation of compatibility between versions `0.x` and `0.y`. For example, a processor must not activate implementation `0.4` to satisfy a requested version of `0.2`.
+
+### Ordering
```

Comment:
I'm not sure what the purpose of this section is. I see it is referenced once in an "as a practical matter" setting but I don't even think that line makes sense — since our logic reasonably treats major version bumps as always backwards-incompatible (requiring processors to enumerate all major versions they support), I don't really know why it particularly matters that you choose the lowest major version. I'd say as a practical matter you should choose a major version you believe to be supported by the processors you care about and the minimum minor version within that — which isn't necessarily a particularly old major version! (Note that this sections automatically goes away if by some chance you agree with me that we should remove major versions from the spec.)

Reply:
I'd like to raise the above point again: nothing in the spec depends on the semantics of ordering, so can we just leave it out?